app/Controller/OAuthController.php

```diff
@@ -127,4 +127,20 @@ protected function authenticationFailure($message)
             'title' => t('Login')
         )));
     }
+
+    /**
+     * Redirect the user after the authentication
+     *
+     * @access private
+     */
+    private function redirectAfterLogin()
```

Comment:
Please, do not duplicate code.

Move this method from `AuthController` to the `BaseController` and make it protected.

Reply:
done